build-packages/webpack-i18n-runtime/build-config/config.plugin.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-param-reassign */
+const path = require('path');
+const WebpackI18nTracker = require('./webpack-i18n-tracker');
+
+// Provide an additional loader directory from this module
+function addResolveLoader(config) {
```

Comment:
Any reason why functions are used instead of function assignments? Like `const addResolveLoader = (config) => { ... }`?

Reply:
That seemed cleaner to me - no reason to bind context here. 
I have refactored this to assignment expressions to maintain a unitary coding style with other packages.